tasks/52_sum-all-numbers-in-range/sum-all-numbers-in-range.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Calculates sum of two numbers and all numbers between them
+ * @example
+ * // returns 45
+ * sumAll([5, 10])
+ * @param {Number[]} arr - array of two numbers
+ * @return {number} - sum of two numbers and all numbers between them
+ */
+export function sumAll(arr) {
+  let sum = 0;
+  for (let i = Math.min(...arr); i <= Math.max(...arr); i++) {
```

Comment:
Math.min and Math.max will be calculated on every iteration. I'd suggest to move it to a variable